### PR TITLE
Add Logging Settings datasources.

### PR DESCRIPTION
**Pull request by zachberger**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:new-datasource
`google_logging_organization_settings`
```

```release-note:new-datasource
`google_logging_folder_settings`
```

```release-note:new-datasource
`google_logging_project_settings`
```

### CODE REVIEW

**Comment by zachberger** [comment]
Any insight to why `TestAccLoggingOrganizationSettings_datasource` did not run?

**Comment by shuyama1** [comment]
> Good question! I'm not entirely sure- I thought we might have been missing an env var & skipped the test, but that doesn't seem to be the case. @shuyama1 can you tell?

Looks like it got passed in the replaying mode. I wonder if we run the actual acceptance test instead of in VCR during the replaying mode for this test specifically. I'll need to dig deep into it to find out what happens exactly. Meanwhile, I triggered an independent run for `TestAccLoggingOrganizationSettings_datasource` on TC and it passed: https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_GoogleBeta_MmUpstreamTesting_GOOGLEBETA_PACKAGE_LOGGING/64618?buildTab=tests

File: mmv1/third_party/terraform/services/logging/data_source_google_logging_folder_settings.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+				Description: `The folder for which to retrieve settings.`,
+			},
+			"disable_default_sink": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: `If set to true, the _Default sink in newly created projects and folders will created in a disabled state. This can be used to automatically disable log storage if there is already an aggregated sink configured in the hierarchy. The _Default sink can be re-enabled manually if needed.`,
+			},
+			"kms_key_name": {
+				Type:     schema.TypeString,
+				Optional: true,
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/logging/data_source_google_logging_folder_settings.go:27]: I think this is Computed, given this is a datasource? It's not required for a getSettings call: https://cloud.google.com/logging/docs/reference/v2/rest/v2/folders/getSettings

(same for both other levels)

**Review by rileykarson — COMMENTED** [review]
We'll need to add website files as well: https://googlecloudplatform.github.io/magic-modules/develop/add-handwritten-datasource/#add-documentation.

Otherwise, generally looks good- one comment inline.

**Comment by zachberger** [comment]
Thanks @rileykarson I've fixed up the property and added documentation.

File: mmv1/third_party/terraform/website/docs/d/logging_folder_settings.html.markdown [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+subcategory: "Cloud (Stackdriver) Logging"
+description: |-
+  Describes the settings associated with a folder.
+---
+
+# google\_logging\_folder\_settings
+
+Describes the settings associated with a folder.
+
+To get more information about Service, see:
```
Comment (rileykarson) [inline mmv1/third_party/terraform/website/docs/d/logging_folder_settings.html.markdown:11]:
```suggestion
To get more information about LoggingFolderSettings, see:
```

File: mmv1/third_party/terraform/website/docs/d/logging_organization_settings.html.markdown [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+subcategory: "Cloud (Stackdriver) Logging"
+description: |-
+  Describes the settings associated with a organization.
+---
+
+# google\_logging\_organization\_settings
+
+Describes the settings associated with a organization.
+
+To get more information about Service, see:
```
Comment (rileykarson) [inline mmv1/third_party/terraform/website/docs/d/logging_organization_settings.html.markdown:11]:
```suggestion
To get more information about LoggingOrganizationSettings, see:
```

**Review by rileykarson — APPROVED** [review]
LGTM- will merge once the downstreams regenerate from the suggestions I applied.

**Comment by zachberger** [comment]
I believe this failure is unrelated to my change, however I did experience this same failure in https://github.com/GoogleCloudPlatform/magic-modules/pull/9384 which I attempted to resolve with https://github.com/GoogleCloudPlatform/magic-modules/pull/9384/commits/693d1845827bf2b31187a2f0b757ad7c960b120e

**Comment by rileykarson** [comment]
Yeah, that's non-blocking. Confirmed we're not merging a new cassette for that.

**Pull request merged**